wiki/People/Language_Moderators/pt-br.md

```diff
@@ -0,0 +1,38 @@
+A maioria dos canais presentes no osu! têm seus próprios moderadores, que cuidam de tudo que possa invadir ou ir contra a atmosfera do canal. Embora existir vários QATs/GMTs cuidando desses problemas graças aos seus poderes, existem vários OPs da conversa que não fazem parte do QAT que moderam o canal para ajudar os outros moderadores.
```

Comment:
Before this line you forgot to add a heading "Moderadores de Idioma", just like in english.